code/include/playfab/PlayFabClientAuthApi.h

```diff
@@ -0,0 +1,352 @@
+// Copyright (c) Microsoft Corporation
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#if !defined(__cplusplus)
+#error C++11 required
+#endif
+
+#pragma once
+
+#include <playfab/PlayFabAuth.h>
+#include <playfab/PlayFabClientDataModels_c.h>
+
+extern "C"
+{
+
+/// <summary>
+/// <LoginWithAndroidDeviceID documentation goes here.>
```

Comment:
><LoginWithAndroidDeviceID documentation goes here.> [](start = 4, length = 51)

Just to make sure - this is a task for a later pr?

Reply:
Ya. Will open a task so this doesn't get lost.  Not 100% sure how we will provide these docs but I have a couple ideas.

---
In reply to: [607288277](https://github.com/jplafonta/XPlatCSdk/pull/2#discussion_r607288277) [](ancestors = 607288277)